win32/deluge-bbfreeze.py

```diff
@@ -19,11 +18,11 @@
 python_path += os.path.sep
 
 print "Python Path: %s" % python_path
-gtk_root = os.path.join(gtk.__path__[0], "..\\runtime\\")
+gtk_root = "C:\\Python27\\Lib\\site-packages\\gnome"
```

Comment:
This is correct but i do agree that re-writing this to not be hardcoded would be better.